openlibrary/accounts/model.py

```diff
@@ -652,15 +654,16 @@ def audit_accounts(email, password, test=False):
         audit['has_ol'] = email
 
         ia_account = InternetArchiveAccount.get(email=email, test=test)
-
-        # Should get the IA account linked to this OL account, if one
-        # exists. However, xauthn API doesn't yet support `info` by
-        # itemname. Fortunately, we have all the info we need at this
-        # stage for the client to be satisfied.
         if ol_account.itemname:
-            audit['has_ia'] = ol_account.itemname  # XXX should be email
+            audit['has_ia'] = ol_account.itemname
             audit['link'] = ol_account.itemname
-            return audit  # special case; unable to get ia acc
+
+            # This used to allow users to continue logging in with
```

Comment:
Let's make sure this comment describes present reality. And it'd be great if it also described what's about to happen.